wiki/Tournaments/4DM/3/en.md

```diff
@@ -30,10 +30,10 @@ The **4 Digit osu!mania World Cup 3** (***4DM3***) was a worldwide country-based
 
 | Placing | Prize(s) |
 | :-: | :-- |
-| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 6 months of osu!supporter tag for each team member |
-| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter tag for each team member |
-| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter tag for each team member |
-| *4th-8th place* | 1 month of osu!supporter tag for each team member |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 6 months of osu!supporter for each team member |
```

Comment:
All of the other instances I have seen list the unique profile badge at the end, e.g. https://github.com/ppy/osu-wiki/pull/5861/files#diff-2961a8db95da64432a8707145b16847ec16ffa1d3b7a62e7814e1b856f967881R24